src/environments/environment.ts

```diff
@@ -6,5 +6,6 @@ export const environment = {
   production: false,
   profileUrl:
     'https://landville-backend-web-api.herokuapp.com/api/v1/auth/profile/',
-  api_url: 'https://landville-backend-web-api.herokuapp.com/api/v1',
+  // api_url: 'https://landville-backend-web-api.herokuapp.com/api/v1',
+  api_url: 'http://localhost:8000/api/v1',
```

Comment:
You might have left this here for testing purposes. If this gets merged, things will break

Reply:
for now while heroku is down I'm using localhost

Reply:
👍 , just don't forget it there